### PR TITLE
Fixed: Admin Bar overlap issue with main menu.

### LINKED ISSUE #152: Admin bar & menu overlap if screen is between 768-782px wide
The menu's responsive breakpoint is 48em (768px), but core's breakpoint is 782px. At 782px and below the admin bar gets taller. At 767px, the menu stops docking to the top. At any size in between, the menu positioning doesn't account for the taller admin bar, and they overlap:

screen shot 2016-10-06 at 12 25 16 am

### PR DESCRIPTION
**Pull request by Swapnildhanrale**
Fixed: #152 

The default WordPress admin bar height is 46px, and we give top 32px to the menubar we need to give this 46px, means when the menu is fixed then it will get top 46px and not overlap with the admin bar.

### CODE REVIEW

**Comment by Swapnildhanrale** [comment]
Now, the used `px` unit to fix this issue. We'll be able to use unit `em` like as shown below.

```
@media screen and (max-width: 48.8em) and (min-width: 48em) {
    .admin-bar .site-navigation-fixed.navigation-top,
    .admin-bar .site-navigation-hidden.navigation-top {
        top: 46px;
}
```

 But when we use the em CSS then it will not work perfectly.

**Comment by melchoyce** [comment]
Looks like this PR has some conflicts with style.css now.

File: style.css [hunk 1]
```diff
@@ -3338,4 +3338,13 @@ article.panel-placeholder {
...
 	}
+}
+	
+@media screen and (max-width: 782px) and (min-width: 768px) {
+
+    .admin-bar .site-navigation-fixed.navigation-top,
+    .admin-bar .site-navigation-hidden.navigation-top {
+        top: 46px;
+    }
+
```
Comment (ntwb) [inline style.css]: Remove this blank line please

**Review by ntwb — CHANGES_REQUESTED** [review]

**Review by ntwb — APPROVED** [review]

File: style.css (hunk 1 above)
Comment (Swapnildhanrale) [reply style.css]: Removed the wanted space from this line

**Comment by Swapnildhanrale** [comment]
Merged `WordPress:master` branch in `Swapnildhanrale:admin-bar-overlap-issue`. Ready for merge.

**Comment by Swapnildhanrale** [comment]
Merged WordPress:master branch in Swapnildhanrale:admin-bar-overlap-issue. Ready for merge.

File: style.css [hunk 2]
```diff
@@ -3303,3 +3303,10 @@ article.panel-placeholder {
 		margin-right: -20%;
 	}
 }
+
+@media screen and (max-width: 782px) and (min-width: 768px) {
+    .admin-bar .site-navigation-fixed.navigation-top,
+    .admin-bar .site-navigation-hidden.navigation-top {
+        top: 46px;
+    }
+}
```
Comment (laurelfulford) [inline style.css:3312]: Can you please add a new line to the end of the file?

**Comment by laurelfulford** [comment]
@Swapnildhanrale the fix looks good! Can I just get you to add a new line to the very end of the style.css for correct formatting? Thanks!

**Comment by davidakennedy** [comment]
@Swapnildhanrale Thanks for your contribution! What's your GitHub user name, so I can ensure you receive props in WordPress 4.7?

**Comment by Swapnildhanrale** [comment]
Hi @davidakennedy my username is **swapnilld**

**Comment by Swapnildhanrale** [comment]
Hi @davidakennedy I am extremely sorry. I have given the wrong username, my username is  [https://profiles.wordpress.org/swapnild](https://profiles.wordpress.org/swapnild). It is my first time in the contribution of WordPress can you please update my name in core contributors list.

**Comment by davidakennedy** [comment]
Hi @Swapnildhanrale! I'm really sorry about that! Typos are easy to make, especially if you're excited about your first contribution. 😄 

It looks like you can be added to the credits page retroactively. See: https://wordpress.slack.com/archives/core-committers/p1481216033000020

So we'll fix it soon! Thank you for your contribution!

**Comment by davidakennedy** [comment]
Fixed here: https://meta.trac.wordpress.org/changeset/4507

cc: @Swapnildhanrale

**Comment by ocean90** [comment]
FYI: Credits are cached in several places so it can take up to 24 hours until it's visible in all lists.

**Pull request merged**